### DISCUSSION TITLE
In Class exercise - Feb 13 (abstract review)

### DISCUSSION THREAD

**Discussion by sr320** [course-fish521-2023/assistance#7] [Show and tell]
Provide feedback on the following abstract:

>Owing to a number of concerns about climate change, we set out to understand how fish respond to warming water temperature. Using a collection of mesocosms set out in a large field near our research station located in the central United States, we compared fish growth rates under ambient temperature compared to elevated temperatures indicative of a possible warming climate regime. This was important because fish will be even more important as a food source in the future as the number of humans on earth continues to grow and will ultimately exceed the carrying capacity of traditional land-based agricultural systems and evidence suggests already that aquaculture may be a necessary and readily available source of important protein. Our experimental results were compared to experimental controls using a 2-way ANOVA using post-hoc Tukey tests. We discovered that warmer temperatures of water in the tanks was pretty significant because our p-values were generally less than would be expected by chance alone, except in two cases where they were really close to 0.1, which is still pretty low but perhaps greater than some people have suggested is a critical threshold for statistical validity. Owing to the importance of our results, we now know that water temperatures in the future will be warmer and therefore fish may change in the future as well. Clearly, our extraordinary findings have huge implications for the future of fisheries management for a healthy and sustainable planet.

**Comment by cvaage** [comment]
The introductory sentence is too broad and does not provide the reader with enough detail to interpret the likely nuance within the paper. I would recommend narrowing down the sentence and go a level deeper to elaborate on a widely applicable prompt to research.  The description of the methods is written in a rather informal way and could use more technical descriptors and specificity concerning the study location. The second sentence would benefit from grammatical review as well for clarity. The warrants for this study are well described, but done so in a manner that would be difficult for the reader to understand. Consider revising the third sentence's structure. The statistical methods are well-described, though the analyses are not well explained. Use of language, such as "pretty significant," is poor writing and should be removed or rephrased to properly explain the results. Good phrasing in the concluding sentences that broaden implications for the audience and field as a whole, but could be done so in a manner that is less self-praising.

**Comment by n-grace-henry** [comment]
It would be helpful to the reader in understanding the context of this research to have some background sentences earlier in the abstract. Some background is given when it is explained why this research is important, but this is after the study has already been mentioned, starting with background would give more clarity and flow. 
The sentence starting with "We discovered that warmer temperatures of water..." is a bit long and this makes it difficult to follow. This could be broken up into multiple sentences, but also there might be a better way to phrase these results. Lots of emphasis is given to the two cases which were close to 0.1, when emphasis should instead be placed on the majority of the cases that were shown to be significant. Changing the phrasing to get rid of "pretty significant" would also strengthen this case to make the language more clear and conclusive. 
In the second to last sentence it is stated that "we now know that water temperatures in the future will be warmer",  however this is not the goal that was previously stated earlier in this abstract. The goal of this study seems to be how fish react to changing temperature, not how temperature will change, so this should be made clear.

**Comment by bzhang111** [comment]
The first thing I noticed was that the abstract's sentence structure needed improvement.  Conciseness is important to make the paper easier to read.  Some ideas need more detail as well with explanations to back them up.  First person can be removed from the abstract.  There are run on sentences and some repetitiveness.  Also, the abstract should not express any opinions (ex: use of the word "extraordinary").  Also, it should be more general and not contain detailed information regarding statistical tests.  The author can just state their general results and interpretation.

**Comment by EveMHallock** [comment]
While the general topic is conveyed and the abstract is readable, the overall tone is informal and vague. I would prefer specifics about species, methods, locations, and dates in the first and second sentences. In the second half, many adjectives and phrases can be cut, such as "extraordinary", "really close", and "pretty". The sentences starting with "Using a", "This was", and "We discovered", are too long and need to be simplified and clarified. The results can be directly included in a concise way with p-values and standard error or another appropriate method. Their results do not inform warming temperatures, so the concluding statements need to be more accurate.

**Comment by valeste** [comment]
- The author describes their experimental design as exposure to "ambient temperature" and "elevated temperature". It would be better if they described the average temperatures they used for those conditions.
- The importance of fish as a food source aspect should be removed from the abstract and left solely for the introduction.
- The decription of the exact statistical analysis conducted is unecessary in the abstract.
- "We discovered that warmer temperatures of water in the tanks was pretty significant because our p-values were generally less than would be expected by chance alone, except in two cases where they were really close to 0.1, which is still pretty low but perhaps greater than some people have suggested is a critical threshold for statistical validity." This specific sentence is too long and too casual for the context of an abstract. Because of those two aspects, there is a lack of clarity in the points the author is attempting to get across. 
-  "Clearly, our extraordinary findings" in the final sentence of the abstract oversells the importance of their results. I think a connection to how their research builds upon past work and is a novel can be good, but is best saved for the discussion section. 
- The author could improve their abstract by focusing on the experimental design and major findings. This means exclude background information that should be put in the introduction and conclusions regarding the significance of their findings.

**Comment by sr320** [comment]
Please elaborate on / or be explicit on "a number of concerns"

The second sentence could be more precise if it was not so bad. (j/k)

The next sentence "This was important .... " would be be best in the beginning to set-up the rationale for the experiment. 

The methods sentence is not needed.

Pretty significant? really?

End is vague and exaggerated.

**Comment by nateredon** [comment]
- The introductory sentence is very broad, it might be more appropriate to discuss what you specifically did and with what species.
- The study area is very broadly described (central United States). More detail would help the reader understand the context.
- Sentence three should be condensed; the benefits of aquaculture can be explored more thoroughly in the background. It is also a bit of a jump to say that fish will be even more important as a food source just because there are more humans. 
- Following standard guidelines for reporting statistical results would help make the significance of the results presented more apparent and more concise.
- The final sentence oversells the significance and importance of the results, dialing back the language would be a bit more professional.

**Comment by kinmic** [comment]
The results presented here are of scientific interest and appropriate for this journal. However, we feel that while the science is sound and the research seems well executed, more work is required on structure of the manuscript. Additionally, we believe that some further study of the literature in this area would be beneficial to the author.

Sentence 1 - Please give some examples of which concerns related to climate change you are referring to.
Sentence 1 -  What environment? Marine, freshwater, estuarine.
Sentence 2 - Offer an explanation of mesocosm. 
Sentence 2 - Quantify large and specify the location of the field ( preferably by latitude and longitude).
Sentence 2 - Consider separating into multiple sentences.
Sentence 3 - What are the elevated  temperatures? Why those temperatures?
Sentence 4 - Verbose, could be more concise.
Sentence 5 - Significance is is binary, it is or is not.
Sentence 5 - The significance should be reported with the group and temperature it applies to.  Critical thresholds should be addressed in a separate sentence, with appropriate citation.
Sentence 6 - The result references shows a significant change in fish growth rate, however it does not support any guaranteed future climatic conditions. Please revise this conclusion. 
Sentence 7 - Superfluous language ( "extraordinary",  "huge") revise.
Sentence 7 - If you believe this research has management implications, please use the literature to outline how, and what they are.

**Comment by KatLeigh11** [comment]
Dear Author,

It was a pleasure to read your writing. I would like to commend you for writing such a through abstract. I provide the following suggestions below for you to consider:
- this sentence is quite long. I would break it into two sentences. I also would avoid repeating the same word (important), as well as using the phrase "as the Earth's population continues to increase" as it is more concise. "This was important because fish will be even more important as a food source in the future as the number of humans on earth continues to grow and will ultimately exceed the carrying capacity of traditional land-based agricultural systems and evidence suggests already that aquaculture may be a necessary and readily available source of important protein.
- the transition from the above sentence to your results comparison seems a bit abrupt, especially when so little has been stated discussing the analytical methods used in your study. Please include a description of your methods prior to the statement on results comparison.
- I am confused about the statement regarding tanks since this study mentions being located in a field research station. Please explain the set up a bit more.
- You conclusions regarding statistical significance are worded oddly and rather unclear. I would replace them with more descriptive, scientific language.
- I am unsure how your results confirm as fact that water temperatures in the future will be warmer; can you re-phrase?

Best of luck on your revisions!

**Comment by KellyAnnNeal** [comment]
First sentence/ second sentence: connect how the specific location of your research is tied to the broader understanding of temperature effects on fishes across varying ecosystems. Climate IS warming, so eliminate word possible. Used active voice, not passive (is versus was). Break up run on sentences. Put actual P-value in parenthesis if you are going to reference it as significant. Have clearer implications of findings (fish will change is not specific enough).

**Comment by giojacuzzi** [comment]
The abstract fails to provide a coherent motivation for the research, and lacks essential information regarding the study design, methodology, and analysis of results. For example, which fish were studied, where, and under what experimental conditions? The text lacks enumeration of experimental parameters and contains several phrases such as "pretty significant" or "really close". It needs definite statements to clarify the science. Finally, the text overstates the significance of the findings, suggesting implications for fisheries management without providing any, and draws conclusions orthogonal to or beyond the scope of the study regarding behavior of all fish and our understanding of warming due to climate.

**Comment by scuba-sarah** [comment]
The authors can briefly state concerns of global warming in the beginning to give the ready more context. This can be combined with the sentence that state why the study is important. Alternatively this sentence can be moved to the end and discuss why the findings are important.
I think it would be beneficial to at some point clarify the specie(s) used for experimentation. An assemblage of local species to the central US or a single species?
Results can be summarized in a more succinct way. Removing words like “pretty” will be helpful.
The last two sentences can be re-written to directly state implications of the study rather than stating that there are implications.

The abstract has all the required components (intro, questions, results, the so-what) and it is a great foundation. With additional clarification and removal of extraneous words, the abstract will be ready for publishing.

**Comment by KalenaKdB** [comment]
The abstract is too lengthy and a lot of the information shared could be reserved for the full paper. Also, the structure of this paper would significantly improve with better proofreading and wordsmithing. There is more than one instance of a run-on sentence, as well as multiple uses of the same adjective in one sentence. The authors would benefit from being more concise and to the point as well. The study methodology was well described but the analyses portion isn't descriptive enough for an abstract.

**Comment by themacfreezie** [comment]
Abstract in normal type-face, _comments in italics._

Owing to a number of concerns about climate change, we set out to understand how fish respond to warming water temperature.
_This is all somewhat non-specific, especially the reference to 'fish'. Fish refers to an extremely diverse group of species and more specificity would lend validity to the paper's premise._

Using a collection of mesocosms set out in a large field near our research station located in the central United States, we compared fish growth rates under ambient temperature compared to elevated temperatures indicative of a possible warming climate regime. 
_Again, lots of ambiguity here. Again, greater specificity would lend weight to the study's importance. Where is this research taking place? What are some species included in your mesocosms? What temperature ranges are being considered? While it is not possible to include full detail in an abstract some choice information would be helpful._

This was important because fish will be even more important as a food source in the future as the number of humans on earth continues to grow and will ultimately exceed the carrying capacity of traditional land-based agricultural systems and evidence suggests already that aquaculture may be a necessary and readily available source of important protein. 
_This sentence is long and could probably broken up. The verb tense also seems mismatched in places, which could benefit from the passage being split into two or more sentences. Additionally, it contains a number of assertions that may be left to discuss in detail in the body of the work, rather than being stated without support in an abstract._

Our experimental results were compared to experimental controls using a 2-way ANOVA using post-hoc Tukey tests. 

We discovered that warmer temperatures of water in the tanks was pretty significant because our p-values were generally less than would be expected by chance alone, except in two cases where they were really close to 0.1, which is still pretty low but perhaps greater than some people have suggested is a critical threshold for statistical validity. 
_There are a lot of strange normative statements here. It might be best to refrain from the use of terms such as 'pretty significant', 'really close', 'pretty low', 'some people have suggested', and the like. _

Owing to the importance of our results, we now know that water temperatures in the future will be warmer and therefore fish may change in the future as well. 
_Consider rephrasing. Has your research shown that temperatures will be warmer in the future?_

Clearly, our extraordinary findings have huge implications for the future of fisheries management for a healthy and sustainable planet.
_This statement reads as a bold and as yet unfounded assertion. It would be best to seriously tone down the self-congratulatory rhetoric._

**Comment by cskalley** [comment]
Overall, this is an interesting study that can inform future patterns in fish growth as the climate changes. That being said, I think that some of the sentences could use some tightening. Generally, I would avoid colloquialisms like "pretty significant," check for subject/verb agreements (for ex: "warmer temperatures of water in the tanks WERE significant," and address sentence structure issues. My general rule of thumb is that if you are using the word "and" twice in a sentence, you should consider reworking that sentence.   

Try to avoid overstating the importance of your findings by avoiding words like "extraordinary"  and "huge."  Also, it might be best to avoid introductory clauses that don't accurately set up the rest of the sentence.  For example, you say "owing to the importance of our results, we now know that water temperatures in the future will be warmer and therefore fish may change in the future as well." Here, "Owing to the importance of our results" doesn't seem to fit with this sentence, since the warming water temperature was a finding that was independent from your study.

**Comment by lizboggs** [comment]
This abstract is quite vague and clunky to read; while climate change is a familiar topic to many, the phrases used here (e.g. "a number of concerns", "pretty significant", etc.) don't really inform the reader about the specific threats to fish that climate change brings or any specifics on the fish that the study was investigating. There is a distinct lack of quantitative evidence provided for the experimental design or the resulting data that is briefly mentioned. Given that the first two-thirds of the abstract doesn't provide much information on the background of this study or its importance, I hesitate to accept that a single experimental run provides "extraordinary" findings as is said in the final sentence.
